### DISCUSSION TITLE
is Ray Tracing a possibility ?

### DISCUSSION THREAD

**Discussion by XDM-Inc** [minecraft-linux/mcpelauncher-manifest#923] [General]
i see the option for ray tracing is grayed out. i do lave a 3090 ti.

**Comment by connorjlbrown** [comment]
I am also wondering if this might be a limitation with the unofficial launcher. I have a Geforce GTX 750 which is quite an old card but should be at least enough to run the mobile version of the deferred technical preview. It is probably a lot more complicated than it immediately seems
